docs/reference/field_types.rst

```diff
@@ -259,3 +259,40 @@ Parameter                   Description
             ])
         ;
     }
+
+Create your own field type
+--------------------------
+
+Field types are Twig templates that are registered in the configuration section matching your ORM provider, which is one of:
+
+1.  ``sonata_doctrine_orm_admin``
+2.  ``sonata_doctrine_phpcr_admin``
+3.  ``sonata_doctrine_mongodb_admin``
+4.  ``sonata_propel_orm_admin``
```

Comment:
```suggestion
Field types are Twig templates that are registered in the configuration section matching your model manager.
```

Reply:
Model managers should remain decoupled from this bundle. Sonata provides some common implementations (propel is deprecated), but you can even provide your own model manager.

Reply:
How do you suggest I write that down in the documentation?

Reply:
The way I think it should be written is as I suggested (omitting the list of model managers currently provided by Sonata packages, keeping the example with "sonata_doctrine_orm_admin" like you already have done).

Reply:
My apologies, I overlooked the green part of your suggestion.

Reply:
Thank you! :+1: